docs/examples/containerized_heketi_dedicated_gluster/README.md

```diff
@@ -0,0 +1,669 @@
+	# ssh-copy-id -i /usr/share/heketi/keys/heketi_key.pub root@gluster26.rhs <3>
+	# chmod 770 heketi* <4>
+```
+<1> Change directory to the dir that was created earlier
+
+<2> Generate the ssh key
+
+<3> copy each ssh key to the dedicated gluster cluster nodes
+
+<4> Change the permissions on the keys
+
+
+#### Obtain and edit the Heketi configuration file to setup the SSH executor.
+
+The heketi.json file is used to configure Heketi on initial container start up. There are multiple
+ways to create and have Heketi properly access and load this file, for this
+example we are simply creating the file and putting in our /usr/share/heketi working directory on each of our nodes.
+Alternatively you could create a secret and configmap to pass into the Heketi container or even store on another share
+point (GlusterFS volume or NFS, etc...).
+
+
+Below is an excerpt from the heketi.json file highlighting the sections
+that potentially could/would need to be changed for your environment.
+
+```
+	{
+  	"_port_comment": "Heketi Server Port Number",
+  	"port": "8888",  <1>
+	...
+        ...
+	"executor": "ssh",  <2>
+
+	"_sshexec_comment": "SSH username and private key file information",
+	"sshexec": {
+  	  "keyfile": "/etc/heketi/keys/heketi_key",  <3>
+  	  "user": "root",  <4>
+  	  "port": "22",  <5>
+  	  "fstab": "/etc/fstab"  <6>
+	},
+        ...
+        
+```
+<1> Default port is 8080, you can configure a different port like 8888
+
+<2> Change "executor": from mock to ssh
+
+<3> Add in the public key directory specified in previous step
+
+<4> Update SSH user (should have sudo/root type access)
+
+<5> Set the port to 22 - remove all other text
+
+<6> Set the fstab to default (etc/fstab) - remove all other text
+
+
+#### Create the dedicated GlusterFS cluster endpoints and service.
+
+The endpoints and service are used to access the GlusterFS cluster to establish mounts.
+These are used for our ReplicationController/Container in future steps and also can be reused
+for any other pods that need access to GlusterFS cluster.
+Run the following yaml examples.
+
+glusterfs-endpoints.yaml
+```
+apiVersion: v1
+kind: Endpoints
+metadata:
+ name: glusterfs-cluster
+subsets:
+ - addresses:
+   - ip: 192.168.1.205
+   ports:
+   - port: 1
+     protocol: TCP
+ - addresses:
+   - ip: 192.168.1.206
+   ports:
+   - port: 1
+     protocol: TCP
+
+```
+
+gluster-service.yaml
+```
+kind: Service
+apiVersion: v1
+metadata:
+  name: glusterfs-cluster
+spec:
+  ports:
+  - port: 1
+```
+
+
+#### Install the Heketi Container and Service.
+From the master node create the following yamls:
+
+
+heketi-service.yaml
+```
+kind: Service
+apiVersion: v1
+metadata:
+  name: heketi
+  labels:
+    name: heketi-service
+  annotations:
+    description: Exposes Heketi service
+spec:
+  type: NodePort
+  selector:
+    name: heketi
+  ports:
+  - name: heketi
+    port: 8888
+    targetPort: 8888
+
+```
+
+heketi-rc.yaml
+```
+apiVersion: v1
+kind: ReplicationController
```

Comment:
Well, since heketi is basically stateless in this case, as we are storing the heketi.db on our actual Gluster server, I didn't see a need for this simple kick-the-tires demo to use a daemonset or deployment.  However, I could see the value in being consistent with the existing deploy yaml files, would you prefer I switch to a deploymentconfig for heketi?